src/components/TileCategorySelection/index.jsx

```
@@ -0,0 +1,143 @@
+          id="group-filter"
+          value={validGroupFilter}
+          label={t('group')}
+          onChange={handleGroupFilterChange}
+          slotProps={{
+            input: { 'aria-label': t('group') },
+          }}
+        >
+          {uniqueGroups.map((group) => (
+            <MenuItem key={group} value={group}>
+              {mappedGroups &&
+              mappedGroups[gameMode] &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)